conf/modules.config

```diff
@@ -992,11 +993,13 @@ process {
     }
 
     withName: KRAKENUNIQ_PRELOADEDKRAKENUNIQ {
+        tag = { ${meta.single_end} ? "single_end" : "paried_end" }
```

Comment:
does not correctly parse meta.single_end despite working within the module itself...